less/_utilities.less

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) 2019 Eclipse Foundation, Inc.
```

Comment:
Are those styles something we could use in other sites?

Also your copyright says 2019.

Reply:
Yeah we could definitely use them elsewhere. I think it should be fine to merge for now because the issue seems to be a high priority. Unless you think otherwise.

I'll create an issue to clean this up (maybe other sites too) and add them to solstice-assets. At the same time, I want to add a few other utility classes we could use. 

I'll update the copyright header.

Reply:
Ok sounds good to merge now and create an issue to clean up.